src/ui/components/ShowMoreCard/index.js

```diff
@@ -11,8 +11,8 @@ import Card from 'ui/components/Card';
 
 import './ShowMoreCard.scss';
 
-
-const MAX_HEIGHT = 100;
+// This refers to height of Card-contents class, beyond which it will add read more link
```

Comment:
I would write:

    `.Card-contents` CSS class

A period is missing at the end of the sentence, which should be wrapped to 80 chars too.